bspump/declarative/expression/utility/mapexpr.py

```diff
@@ -36,12 +35,34 @@ def __init__(self, app, *, arg_what, arg_in, arg_else=None):
 			self.Default = arg_else
 
 		self.Map = arg_in
+		self.MapNormalized = dict()
+		self.OutletType = None  # Will be determined in `initialize()`
 
+	def set(self, key, value):
```

Comment:
?

Reply:
@ateska The same as above. Expressions in lists or dicts do not pass optimization.